ERP-BACKEND/app/routers/crm.py

```
@@ -163,7 +78,7 @@ def get_company(company_id: int, db: Session = Depends(get_db), current_user = D
     return company
 
 @router.put("/companies/{company_id}", response_model=CompanyResponse)
-def update_company(company_id: int, data: CompanyCreate, db: Session = Depends(get_db), current_user = Depends(get_current_user)):
+def update_company(company_id: int, data: CompanyUpdate, db: Session = Depends(get_db), current_user = Depends(get_current_user)):
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)